common/resolver.go

```diff
@@ -127,13 +127,13 @@ func (r *resolverImpl) cachePut(key string, value string) {
 // so names like /./. aren't valid
 
 // PathRegex regex for destination path
-var PathRegex = regexp.MustCompile(`^/[\w.]*[a-zA-Z][\w.]*/[\w.]*[a-zA-Z][\w.]*$`)
+var PathRegex = regexp.MustCompile(`^/[\w.]*[[:alnum:]][\w.]*/[\w.]*[[:alnum:]][\w.]*$`)
```

Comment:
Do you have pure numeric path components? /w already allows a-z, A-Z, 0-9, including the _ (underscore), so it seems like you can have a bunch of numbers as long as you have at least one letter with the current system.